### ISSUE TITLE
Activity Feed 

### ISSUE THREAD

**Issue by SpiritWolfie** [SocialEngine/phpv4-feature-requests#115]
Just a small suggestion, but I know the wall uses a char limit, No one knows what that char limit is when they are typing, so it'd be great to know how many chars people have left to type

This should also be affected if we update the limit our selves and the code should reflect any new increase or decrease we choose to make.

**Comment by DonnaScriptTechs** [comment]
Similar to Twitter? 

Are you also asking for a setting to control the character limit? (we might need to add an "up to" to avoid people putting unlimited and breaking their sites if we add this setting)

**Comment by SpiritWolfie** [comment]
Yes like Twitter, As it stands theirs no way for someone to know when they are about to hit a char limit, so they will type X amount and post it and realize a ton of their stuff is missing, I've had this happen many times..

It'd be nice to have a option in the admin panel to simply adjust the limit (Yes I agree with having a hard limit) but just having a info box of "X amount of chars remaining" that gets updated if you change that up or down would be great
